src/scripts/app.js

```diff
@@ -1 +1,22 @@
 console.log('hello');
+
+Array.from(document.querySelectorAll('.stars')).forEach(function(a) {
```

Comment:
To też poprawiałeś w [TASKU](https://github.com/Varenthein/wdp-1902-03/pull/23).

Zaciągnij sobie aktualnego mastera, bo chyba masz mastera, który jest nieaktualny i wyskoczy Ci pełno konfliktów.